src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -939,6 +939,11 @@ theorem not_def {x : BitVec v} : ~~~x = allOnes v ^^^ x := rfl
   ext i
   simp
 
+@[simp]
+theorem not_not {b : BitVec w} : ~~~(~~~b) = b := by
```

Comment:
Can you please remove the theorem at https://github.com/leanprover/lean4/blob/master/src/Std/Tactic/BVDecide/Normalize/BitVec.lean#L144 and tag this one with `bv_normalize` there as seen here: https://github.com/leanprover/lean4/blob/master/src/Std/Tactic/BVDecide/Normalize/BitVec.lean#L63